src/App.js

```diff
@@ -34,8 +38,24 @@ function App() {
           <Route path="/signup" element={<Signup />} />
           <Route path="/look" element={<Look />} />
           <Route path="/record" element={<WithLogin children={<Record />} />} />
-          <Route path="/create" element={<WithLogin authority="ROLE_TEACHER" children={<Create />} />} />
-          <Route path="/application/:id" element={<WithLogin authority="ROLE_TEACHER" children={<ApplicationManagement />} />} />
+          <Route
+            path="/create"
+            element={
+              <WithLogin authority="ROLE_TEACHER" children={<Create />} />
+            }
+          />
+          {/* Application 페이지 */}
+          <Route path="/application/:id" element={<ApplicationPage />} />
+          {/* Application 관리 페이지 */}
+          <Route
+            path="/application/:id/manage"
```

Comment:
```suggestion
          <Route
            path="/application/:id/manage"
```

이것도 주석 지워주세요
필요없는 주석 지양합시다